probe/pkg/dupedetection/config.go

```diff
@@ -0,0 +1,47 @@
+package dupedetection
+
+import "path/filepath"
+
+const (
+	// DefaultInputDir that dd-service monitors for the new file to generate fingerprints
+	DefaultInputDir = "input"
+	// DefaultOutputDir that dd-service uses to put new fingerprints
+	DefaultOutputDir = "output"
+	// DefaultDataFile is path to the SQLite database used by dd-service
+	DefaultDataFile = "dupe_detection_image_fingerprint_database.sqlite"
+)
+
+// Config contains settings of the dupe detection service
+type Config struct {
```

Comment:
I think this config should be used in the tensor structure in tensor.go. And it should be passed through the supernode app when it starts.